libs/barista-components/breadcrumbs/src/breadcrumbs-item.ts

```diff
@@ -20,11 +20,11 @@ import { Directive, ElementRef } from '@angular/core';
  * A breadcrumbs item that can be used within the `<dt-breadcrumbs>`.
  *
  *  @example
- *  <a dtBreadcrumbsItem href="/hosts">Hosts</a>
+ *  <a DtBreadcrumbsItem href="/hosts">Hosts</a>
  */
 @Directive({
-  selector: 'a[dt-breadcrumbs-item], a[dtBreadcrumbsItem]',
-  exportAs: 'dtBreadcrumbsItem',
+  selector: 'a[dt-breadcrumbs-item], a[DtBreadcrumbsItem]',
```

Comment:
```suggestion
  selector: 'a[dt-breadcrumbs-item], a[dtBreadcrumbsItem]',
```
The selector should still be a[dtBreadcrumbsItem]

Reply:
Please revert in all files and comments.